resources/Materials/TestSuite/stdlib/nodegraph_inputs/nodegraph_multioutput.mtlx

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <nodegraph name="green_graph" >
+    <input name="in" type="color3" value="0, 1, 0" />
+    <separate3 name="green_node" type="multioutput" nodedef="ND_separate3_color3" >
+      <input name="in" type="color3" interfacename="in" value="0, 1, 0"  />
+      <output name="outr" type="float" />
+      <output name="outg" type="float" />
+      <output name="outb" type="float" />
+    </separate3>
+    <output name="green_output" type="float" nodename="green_node" output="outg" />
+    <output name="blue_output" type="float" nodename="green_node" output="outb" />
+  </nodegraph>
+
+  <!-- Test to pick the appropraite output from the nodegraph. In this case this is the green output which is 1.0 or white -->
```

Comment:
Add 2 shaders to route from G, and B which would not work before since it would ignore the "output" attribute names on nodegraph outputs and default to using the node and hence by default use the first output which is "outr".